common/src/main/java/com/revenuecat/purchases/common/CustomerInfoFactory.kt

```diff
@@ -47,7 +52,7 @@ object CustomerInfoFactory {
 
         val entitlements = subscriber.optJSONObject("entitlements")
 
-        val requestDate = Iso8601Utils.parse(body.getString("request_date"))
+        val requestDate = overrideRequestDate ?: Iso8601Utils.parse(body.getString("request_date"))
```

Comment:
This was a bit awkward... We could make it so if the response is missing the request time header, we fail but I thought I would copy what iOS is doing and just fallback to the request date in the body. Happy to discuss this though.

Reply:
I think this makes sense.